modules/theme/src/definitions/globals/product.less

```diff
@@ -73,6 +75,140 @@ body {
+        display: flex;
+        text-align: center;
+        align-items: center;
+        justify-content: center;
+        &.inline {
+            display: inline-block;
+        }
+        &.transparent {
+            background-color: transparent;
+        }
+        &.relaxed {
+            padding: @iconWrapperPadding;
+            &.very {
+                padding: @iconWrapperVeryPadding;
+            }
+        }
+        &.bordered {
+            border-width: @iconWrapperBorderWidth;
+            border-style: solid;
+            border-color: @iconWrapperBorderColor;
+        }
+        &.rounded {
+            border-radius: @defaultBorderRadius;
+        }
+        &.default {
+            svg.icon {
+                path {
+                    fill: @iconDefaultBgColor;
+                }
+            }
+        }
+        &.mini {
+            svg.icon {
+                width: 35px;
+                height: auto;
+            }
+            &.square {
+                svg.icon {
+                    height: 35px;
+                }
+            }
+        }
+        &.tiny {
+            svg.icon {
+                width: 80px;
+                height: auto;
+            }
+            &.square {
+                svg.icon {
+                    height: 80px;
+                }
+            }
+        }
+        &.small {
+            svg.icon {
+                width: 150px;
+                height: auto;
+            }
+            &.square {
+                svg.icon {
+                    height: 150px;
+                }
+            }
+        }
+        &.medium {
+             svg.icon {
+                 width: 300px;
+                 height: auto;
+             }
+            &.square {
+                svg.icon {
+                    height: 300px;
+                }
+            }
+        }
+        &.large {
+            svg.icon {
+                width: 450px;
+                height: auto;
+            }
+            &.square {
+                svg.icon {
+                    height: 450px;
+                }
+            }
+        }
+        &.big {
+            svg.icon {
+                width: 600px;
+                height: auto;
+            }
+            &.square {
+                svg.icon {
+                    height: 600px;
+                }
+            }
+        }
+        &.huge {
+            svg.icon {
+                width: 800px;
+                height: auto;
+            }
+            &.square {
+                svg.icon {
+                    height: 800px;
+                }
+            }
+        }
+        &.massive {
+            svg.icon {
+                width: 960px;
+                height: auto;
+            }
+            &.square {
+                svg.icon {
+                    height: 960px;
+                }
+            }
+        }
+        &.two-tone {
+            svg.icon {
+                path.primary {
+                    fill: @primaryColor;
```

Comment:
Move it to a new variable in `product.less` and use here and below. ex:- `@iconWrapperTwoTonePrimaryColor`